packages/react-charting/src/components/VerticalStackedBarChart/VerticalStackedBarChartRTL.test.tsx

```diff
@@ -7,9 +7,9 @@ import { IVSChartDataPoint, IVerticalStackedChartProps } from '../../index';
 import { VerticalStackedBarChart } from './VerticalStackedBarChart';
 import { getByClass, getById, testWithWait, testWithoutWait } from '../../utilities/TestUtility.test';
 import { VerticalStackedBarChartBase } from './VerticalStackedBarChart.base';
+import * as utils from '@fluentui/react/lib/Utilities';
```

Comment:
why is this entry needed

Reply:
Yes Atishay we are mocking getRTL function from utilities class.